server/sonar-main/src/main/java/org/sonar/application/config/JdbcSettings.java

```diff
@@ -136,19 +136,15 @@ private static void warnIfUrlIsSet(int port, String existing, String expectedUrl
     if (isNotEmpty(existing)) {
       Logger logger = LoggerFactory.getLogger(JdbcSettings.class);
       if (expectedUrl.equals(existing)) {
-        logger.warn("To change H2 database port, only property '{}' should be set (which current value is '{}'). " +
-          "Remove property '{}' from configuration to remove this warning.",
-          JDBC_EMBEDDED_PORT, port,
-          JDBC_URL);
+        logger.warn("To change H2 database port, only property '" + JDBC_EMBEDDED_PORT + "' should be set (which current value is '{}'). " +
+          "Remove property '" + JDBC_URL + "' from configuration to remove this warning.",
+          port);
       } else {
-        logger.warn("Both '{}' and '{}' properties are set. " +
-          "The value of property '{}' ('{}') is not consistent with the value of property '{}' ('{}'). " +
-          "The value of property '{}' will be ignored and value '{}' will be used instead. " +
-          "To remove this warning, either remove property '{}' if your intent was to use the embedded H2 database, otherwise remove property '{}'.",
-          JDBC_EMBEDDED_PORT, JDBC_URL,
-          JDBC_URL, existing, JDBC_EMBEDDED_PORT, port,
-          JDBC_URL, expectedUrl,
-          JDBC_URL, JDBC_EMBEDDED_PORT);
+        logger.warn("Both '" + JDBC_EMBEDDED_PORT + "' and '" + JDBC_URL + "' properties are set. " +
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2629)